web/templates/register.html

```diff
@@ -2,12 +2,17 @@
 {% load staticfiles %}
 
 {% block content %}
-<h1> Register for an Account</h1>
+<div class="container">
+
+	<h1> Register for an Account</h1>
```

Comment:
indentation is really off here - change your default settings to 2 spaces:

just add these lines to your sublime user prefs:

```
	"tab_size": 2,
	"translate_tabs_to_spaces": true
```